src/main/java/com/google/sps/data/ScavengerHunt.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+// Represents a scavenger hunt.
+public class ScavengerHunt {
+  private ArrayList<HuntItem> items;
+  // For saved scavenger hunts: represents which destination the user is on.
+  private int index;
+  private String city;
+
+  public ScavengerHunt(ArrayList<HuntItem> items, int index, String city) {
+    this.items = items;
+    this.index = index;
+    this.city = city;
+  }
+
+  public void changeIndex(int newIndex) {
```

Comment:
Will it ever be correct for a caller to set this to anything other than the current index + 1? If not, `incrementIndex()` seems better.

Reply:
It seems like this function is never used by the Java servlet; I'll remove this from the class.